### PR TITLE
Reword --exit-policy option

### PR DESCRIPTION
**Pull request by dilyanpalauzov**
Closes https://github.com/containers/podman/issues/15297.

### CODE REVIEW

**Comment by rhatdan** [comment]
You need to sign your commits

git commit -a --amend -s
git push --force

**Comment by rhatdan** [comment]
LGTM
@TomSweeneyRedHat @mheon PTAL

File: docs/source/markdown/podman-pod-create.1.md.in [hunk 1]
```diff
@@ -89,10 +89,10 @@ Set custom DNS search domains in the /etc/resolv.conf file that will be shared b
...
 Set the exit policy of the pod when the last container exits.  Supported policies are:
 
-| Exit Policy        | Description                                                                 |
-| ------------------ | --------------------------------------------------------------------------- |
-| *continue*         | The pod continues running when the last container exits. Used by default.   |
-| *stop*             | The pod is stopped when the last container exits. Used in `kube play`.      |
+| Exit Policy        | Description                                                                                                      |
+| ------------------ | ---------------------------------------------------------------------------------------------------------------- |
+| *continue*         | The pod continues running, by keeping its infra container alive, when the last container exits. Used by default. |
+| *stop*             | The pod (its infra container) is stopped when the last container exits. Used in `kube play`.                     |
```
Comment (mheon) [inline docs/source/markdown/podman-pod-create.1.md.in:95]: maybe (including its infra container) instead?

**Comment by TomSweeneyRedHat** [comment]
LGTM
once the tests are happy
Thanks @dilyanpalauzov !

**Comment by rhatdan** [comment]
Saving the planet, force merging.

**Pull request merged**